CMakeLists.txt

```diff
@@ -253,6 +253,14 @@ target_link_libraries(polyfem PUBLIC hdf5::hdf5)
 include(high_five)
 target_link_libraries(polyfem PUBLIC HighFive::HighFive)
 
+# natsort library
+include(natsort)
+target_link_libraries(polyfem PUBLIC natsort::natsort)
+
+# GLOB library
```

Comment:
Why not the fs in the standard?

Reply:
This does natural sorting of file names. For example:
`1.obj, 3.obj, 11.obj, 2.obj` is sorted to `1.obj, 2.obj, 3.obj, 11.obj` instead of `1.obj, 11.obj, 2.obj, 3.obj` like you would get with standard string sorting.

Reply:
And I don't think there is a glob in the standard library.